loops-and-arrays.js

```diff
@@ -37,6 +45,11 @@ const numbers = [2, 4, 5, 29, 38];
 
 const squares = [30, 2, 8, 24, 11];
 
+for (let i = 0; i < squares.length; i++) {
+  squares[i] = squares[i] * squares[i] * squares[i];
+  console.log(squares[i]);
+}
+
```

Comment:
Zelfde als bij de voorgaande twee, om het exacte antwoord te krijgen zoals dat in de opdracht staat zou je de console.log buiten de for loop kunnen plaatsen en enkel (squares) aanroepen ipv (squares[i]) in de console.log.

Maar de manier van wat de code zou moeten doen en de gewenste uitkomsten kloppen overal wel dus het ziet er goed uit over het algemeen.